docs/experiments-plus/getting-group.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: Getting the Group
+sidebar_label:Getting the Group
+slug: /experiments-plus/getting-the-group
+---
+
+One of the most common questions we get is "How do I check the experiment group in code?"  Checking the experiment group _in code_ is actually an anti-pattern.
+Experiment groups are very useful for understanding what a set of parameters represents in the Statsig console.  Comparing "Sorted Long List" vs "Default Search Results"
+is easier to discuss than trying to understand what the `sorted = true, length = 10` parameters represent.
+
+That being said, in code, its much more powerful to directly check parameters and their values. Parameters are the building blocks of your experiments when coding.
```

Comment:
powerful, but simpler (to think about), i.e. you just need to think about either hardcoding a value, or getting it from statsig